cgi-bin/DW/Console/Command/RenameOpts.pm

```diff
@@ -1,7 +1,8 @@
 #!/usr/bin/perl
 #
 # DW::Console::Command::RenameOpts
-# This module 
```

Comment:
THIS MODULE

I wonder if a linter that looks for bad grammar would be more useful or annoying (maybe not for DW, but in general).